FreeRTOS/FreeRTOS
# Add the ability to do a multi-configuration build to the kernel unit tests

Add the ability to do a multi-configuration build to the kernel unit tests

Description
-----------
Add a model Makefile that is able to build multi tests with multi-configurations
add tests of the form tasks_utest_1.c with FreeRTOSConfig_1.h and tasks_utest_2.c with FreeRTOSConfig_2.h

Test Steps
-----------
make tasks builds with no errors or warnings

Note: must add tasks to the UNITS variable for this to work
```makefile
UNITS := list tasks
```
then run
```bash
$ make tasks
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
